lib/coherence/responders.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Responders do
```

Comment:
Should this module not be namespaced with `Coherence` to avoid conflicts with other potential modules in either the including project or another dependency?